custom/icds_reports/models/aggregate.py

```diff
@@ -177,6 +177,21 @@ class CcsRecordMonthly(models.Model, AggregateMixin):
     anc_abnormalities = models.SmallIntegerField(blank=True, null=True)
     date_death = models.DateField(blank=True, null=True)
     person_case_id = models.TextField(blank=True, null=True)
+    husband_name = models.TextField(blank=True, null=True)
+    lmp = models.DateField(blank=True, null=True)
+    migration_status = models.PositiveSmallIntegerField(blank=True, null=True)
+    where_born = models.PositiveSmallIntegerField(
```

Comment:
Migration for ccs_record monthly table will also happen yeah? @challabeehyv

Reply:
@Rohit25negi i see that css_record_monthly is not managed by migration. Do i add it in the 0050_move_to_django ?